bash_completion.sh

```diff
@@ -273,10 +273,10 @@ _buildtest ()
           esac
         ;;
         profiles)
-          local opts="--help -h list"
+          local opts="--help -h --remove -rm list"
           COMPREPLY=( $( compgen -W "${opts}" -- $cur ) )
           if [[ "${prev}" == "list" ]]; then
-            local opts="--help  --theme --yaml -h -y"
+            local opts="--help  --theme --remove --yaml -h -rm -y"
```

Comment:
remove these words `--remove` and `--rm` from here they dont belong in tab completion in `list`

Reply:
done